about/get-involved.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Get Involved
+---
+
+Keyman is a free and open source software project. Continued development of Keyman depends on the generosity of
+you, our users. There are many ways in which you can get involved; keep reading to find the best way you can
+be a part of the journey:
+
+* [Make a donation](#donate)
+* [Create a keyboard layout for your language](#create-keyboard)
+* [Localize Keyman for your language](#localize-keyman)
+* [Support other users](#support-users)
+* [Test a new version](#test-beta)
+* [Write some code](#write-code)
+* [Improve the documentation](#improve-docs)
+* [Join the team!](#join-team)
+
+## Make a donation {#donate}
+[Click here](/donate) to make a donation to the Keyman project by credit card,
+check, or bank transfer. Your donation will be used to help cover the costs of developing Keyman, including
+paying staff and contractors, server costs and hardware and software purchases.
+
+* [Learn more about donating]($donate.keyman.com/faq)
+* [Donate now](/donate)
+
+## Create a keyboard layout for your language ## {#create-keyboard}
+Keyman Developer is our comprehensive tool for creating keyboard layouts for every major platform; it is of
+course completely free and open source! Go ahead and [download Keyman Developer](/developer/download)
+to start creating keyboard layouts today. Once you have created a layout, you can share it online, and if you like,
+contribute it to the Keyman keyboard repository.
+
+* [Learn about Keyman Developer](/developer)
+* [Share your keyboard layouts]($help.keyman.com/developer/keyboards)
+
+## Localize Keyman for your language ## {#localize-keyman}
+Many of the Keyman products have an option to change the displayed language. In Keyman for Android and
+Keyman for iPhone and iPad, the Keyman UI will match your device's locale (if the translated strings are available).
+If your language is not an available option, or some of the strings need to be updated, visit
+[https://translate.keyman.com](https://translate.keyman.com) to help translate Keyman into your language.
```

Comment:
* translate.keyman.com needs to be added to KeymanHosts!
```suggestion
[translate.keyman.com](https://translate.keyman.com/) to help translate Keyman into your language.
```
Note that we should probably always include a final slash even if referring to the home page.